smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/HTTPResponseBindingOutputGenerator.kt

```diff
@@ -60,6 +62,7 @@ class HTTPResponseBindingOutputGenerator(
                         writer.write("return \$N()", outputSymbol)
                     } else {
                         if (needsAReader(ctx, responseBindings)) {
+                            writer.addImport(SwiftSymbol.make("ClientRuntime", null, SwiftDependency.CLIENT_RUNTIME, emptyList(), listOf("SmithyReadWrite")))
```

Comment:
Like on the error init generator above, this import exposes `.data()` on `HTTPResponse`.